packages/@webex/plugin-cc/test/unit/spec/services/core/aqm-reqs.ts

```diff
@@ -141,21 +141,20 @@ describe('AqmReqs', () => {
           msg: {},
         },
       }));
-
+    
       try {
         const p = await Promise.all([
           req({}),
           new Promise<void>((resolve) => {
             setTimeout(() => {
-              webSocketManagerInstance.dispatchEvent(
-                new CustomEvent('message', {
-                  detail: JSON.stringify({
-                    type: 'RoutingMessage',
-                    data: {
-                      type: 'AgentCtqCancelled',
-                      interactionId: '6920dda3-337a-48b1-b82d-2333392f9905',
-                    },
-                  }),
+              eventBusInstance.emit(
```

Comment:
Had to change the way we mock the event, but other than that most test cases are covered)